examples/conflicts/server.js

```diff
@@ -58,7 +66,7 @@ const resolvers = {
       return greeting
     },
     changeGreetingClient: async (obj, args, context, info) => {
-      if (conflictHandler.hasConflict(greeting, args)) {
+      if (conflictHandler.hasConflict(greeting, args, info)) {
```

Comment:
From the frist glance -1.  Why this is needed?

Reply:
Because we need to add things like operation type (query/mutation) and the operation path (getUser/saveUser) in the Prometheus counter. In order to have them there, we need to pass the resolver info to the conflictListener (aka conflictLogger).
It can definitely be improved but I didn't want to waste time on that as we agreed yesterday